README.md

```diff
@@ -14,3 +14,12 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+Three reasons what asymptotic analysis may be misleading with respect to actual performance in practice are hardware, constant factors, data. When an algorithm is ran on two seperate computers, if one computer has better hardware it will run faster. This can be misleading if the two computers are comparing times. This can be the case for two equal computers as well, just with slightly different parts, there is factors like the memory and CPU that could change results. An algorithm can ignore constant factors when using asymptotic analysis. Larger constant factors can make the algorithm slower for practical input sizes. Something else that can be misleading about asymptotic analyis is if the inputted data into an algorithm is all the same. There can be different types in input of data.
```

Comment:
Not sure about your last reason -- we can consider worst/best/average cases separately, which seems to be what you're getting at.